packages/kbn-alerts-ui-shared/src/rule_type_modal/components/index.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useMemo, useState } from 'react';
+import type { HttpStart } from '@kbn/core-http-browser';
+import type { ToastsStart } from '@kbn/core-notifications-browser';
+import { useLoadRuleTypesQuery } from '../../common/hooks';
+import { RuleTypeModal, type RuleTypeModalProps } from './rule_type_modal';
+import { filterAndCountRuleTypes } from './helpers/filter_and_count_rule_types';
+
+export interface RuleTypeModalComponentProps
+  extends Pick<RuleTypeModalProps, 'onClose' | 'onSelectRuleType'> {
+  http: HttpStart;
+  toasts: ToastsStart;
+  filteredRuleTypes: string[];
```

Comment:
do we have a more specific type for this than `string`?

Reply:
Nope, this is what it is in existing code